docs/examples/request_data/custom_request.py

```diff
@@ -0,0 +1,38 @@
+from litestar import Litestar, Request, get
+from litestar.connection.base import empty_receive, empty_send
+from litestar.enums import HttpMethod
+from litestar.types import Receive, Scope, Send
+
+KITTEN_NAMES_MAP = {
+    HttpMethod.DELETE: "Dziewczynka",
+    HttpMethod.GET: "Pusia",
+    HttpMethod.HEAD: "Nepomuk",
+    HttpMethod.OPTIONS: "Boszko",
+    HttpMethod.PATCH: "Morro",
+    HttpMethod.POST: "Irinka",
+    HttpMethod.PATCH: "Kotel",
+}
```

Comment:
I know I'm going to potentially sound pedantic, but I think this being in non-English is/could be quite jarring for end users reading the documentation.  @litestar-org/maintainers feel free to override, I personally don't mind it I just think maybe it would be a "break" in reading that might be a little much

```suggestion
KITTEN_NAMES_MAP = {
    HTTPStatus.METHOD_NOT_ALLOWED: "Whiskers",
    HTTPStatus.OK: "Mittens",
    HTTPStatus.NOT_FOUND: "Shadow",
    HTTPStatus.RESET_CONTENT: "Boots",
    HTTPStatus.CREATED: "Bella",
    HTTPStatus.ACCEPTED: "Simba",
    HTTPStatus.NO_CONTENT: "Cleo",
}
```

Also either way, I think patch was duplicated